src/rpc/RPCHelpers.hpp

```diff
@@ -744,12 +745,13 @@ decodeCTID(T const ctid) noexcept
  * @brief Log the duration of the request processing
  *
  * @tparam T The type of the duration
- * @param ctx The context of the request
+ * @param request The request to log
+ * @param tag The tag of the context of the request
  * @param dur The duration to log
  */
 template <typename T>
```

Comment:
While you are here, can we change `T` to `Duration` or `DurationType`? `T` is such a generic name.. 🙇